core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5419,6 +5419,21 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey SECURITY_AUTHENTICATION_CUSTOM_SASL_CLIENT_CLASS =
+      new Builder(Name.SECURITY_AUTHENTICATION_CUSTOM_SASL_CLIENT_CLASS)
+          .setDescription(
+              "The class to provide a custom security authentication sasl client handler.")
```

Comment:
Update the descriptions according to the new names?

Reply:
Thanks for the reminding.